spec/next_rails/gem_info_spec.rb

```diff
@@ -1,58 +1,39 @@
-require_relative ".././spec_helper"
-require_relative "../../lib/next_rails/gem_info"
+# frozen_string_literal: true
+
+require "spec_helper"
+
+require "timecop"
 
 RSpec.describe NextRails::GemInfo do
-  describe "#state" do
```

Comment:
@JuanVqz we do not need these specs anymore?  `#state` and `#find_latest_compatible`

Reply:
@julioalucero thank you for catching it! I fixed it already, please feel free to review it when possible, 👍